cmd/awscodecommitsource/Dockerfile

```diff
@@ -1,19 +1,15 @@
 FROM golang:1.14-alpine AS builder
 
-ENV GO111MODULE on
 ENV CGO_ENABLED 0
 ENV GOOS linux
 ENV GOARCH amd64
 
-WORKDIR /go/src/awscodecommit
+WORKDIR /go/src/awscodecommitsource
 
-RUN apk update && apk add --no-cache openssh-client ca-certificates
+RUN apk add --no-cache openssh-client ca-certificates
 
-COPY awscodecommit/go.mod awscodecommit/go.sum /go/src/awscodecommit/
-RUN go mod download
-
-COPY awscodecommit/ /go/src/awscodecommit/
-RUN go install -installsuffix cgo ./...
+COPY ./ .
+RUN go install -installsuffix cgo ./cmd/awscodecommitsource
```

Comment:
> I had to consolidate modules inside a single top-level go.mod 

This would be help clean up reduntant https://github.com/triggermesh/aws-event-sources/blob/master/.circleci/config.yml#L14-L22 bits from the circle ci config with a single `go/mod-download-cached` from the go orb which requires a top level `go.sum`.  :+1: